<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModeMem</class>
 <widget class="QDialog" name="DialogModeMem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {border-image: url(:/image/tipsBkg.png);}
QWidget
{
    font-size:30px;
}
QToolButton
{
	font-size:20px;
    border-image: url(:/image/btnG.png);
    color: rgb(255, 255, 255);
}

QToolButton:pressed
{
    border-image: url(:/image/btnR.png);
}
QPushButton
{
    border-image: url(:/image/btnG.png);
    color: rgb(255, 255, 255);
}

QPushButton:pressed
{
    border-image: url(:/image/btnR.png);
}
QSpinBox
{
        text-align:center;
        color:rgb(255,255,255);
        border-image: url(:/image/btnR.png) no-repeat;
}

QSpinBox::up-button
{
        subcontrol-position:right;
        width:50px;
        height:50px;
        image: url(:/image/upArrow.png);
}

QSpinBox::down-button
{
        subcontrol-position:left;
        width:50px;
        height:50px;
        image: url(:/image/downArrow.png);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="horizontalSpacing">
    <number>20</number>
   </property>
   <property name="verticalSpacing">
    <number>30</number>
   </property>
   <item row="1" column="4">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="radioButton_5">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QRadioButton" name="radioButton_6">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="radioButton_1">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QRadioButton" name="radioButton_7">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆7</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QRadioButton" name="radioButton_8">
     <property name="styleSheet">
      <string notr="true">
QRadioButton:checked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnR.png);
}

QRadioButton:unchecked
{
color: rgb(255, 255, 255);
	border-image: url(:/image/btnG.png);
}</string>
     </property>
     <property name="text">
      <string>记忆8</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonRead">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QLabel" name="labelTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>上界</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
